tests/service/data/SyntaxTree/UnionCase/Missing keyword of.fs

```diff
@@ -0,0 +1,7 @@
+module Module
+
+type U =
+    | Case1
+    | Case2 string
+    | Case3 of int
+    | Case4 4
```

Comment:
Please also add test cases with some member and/or expression below, to ensure that the parser has actually recovered and can continue parsing.